scripts/k-crossgen.cmd

```diff
@@ -2,10 +2,10 @@
 SETLOCAL
 SET ERRORLEVEL=
 
-SET "DOTNET_RUNTIME_PATH=%~dp0."
+SET "KRE_RUNTIME_PATH=%~dp0."
```

Comment:
feels like these scripts should be have find replace tokens __RUNTIME_NAME_ and when we build then we would replace those tokens with the name.